moped-database/docker-compose.yml

```diff
@@ -21,6 +20,7 @@ services:
             - 'HASURA_GRAPHQL_JWT_SECRET={"type":"RS256","jwk_url": "https://cognito-idp.us-east-1.amazonaws.com/us-east-1_U2dzkxfTv/.well-known/jwks.json","claims_format": "stringified_json"}'
             - HASURA_GRAPHQL_ADMIN_SECRET=hasurapassword
             - ACTIVITY_LOG_API_SECRET=hasurapassword
+            - HASURA_GRAPHQL_NO_OF_RETRIES=30
```

Comment:
this was a [TIL](https://hasura.io/docs/2.0/deployment/graphql-engine-flags/reference/#number-of-retries) for me. I was a little confused about the docs saying it was "deprecated" but then I read this blurb.

> In v2.0, the values of the following env vars are used to define the connection parameters of the default database while updating an existing instance **or while starting a fresh instance.**

Reply:
Thanks for finding that Mike, and double thanks for checking so thoroughly into the documentation on the variable. I had not seen the depreciation notice that you pointed out -- I saw the variable name somewhere, thought I'd give it a whirl and it did exactly what I had expected it to do. I don't think they are using the word deprecation here quite right -- they should call it a one-time-use value or something like that. But what I really want to say is - thank you for your detailed review.